.github/workflows/install_deps/action.yml

```diff
@@ -25,7 +25,7 @@ inputs:
   jax_version:
     description: The version of JAX to install for any job that requires JAX
     required: false
-    default: '0.4.23'
+    default: '0.4.26'
```

Comment:
@PietropaoloFrisoni could we instead test with 0.4.28, since this matches the catalyst JAX requirements?

Reply:
Sure. I replied on Shortcut (I'll resolve this conversation)